.circleci/config.yml

```diff
@@ -42,11 +42,13 @@ jobs:
 
       - run:
           name: Check Code Style for ml-agents and gym_unity using black
+          # Some pre-commit hooks (namely mypy) don't work well on the entire repo because of the pacakage structure.
+          # So run it on all the files of the subdirectory instead.
           command: |
             . venv/bin/activate
-            black --check ml-agents
-            black --check ml-agents-envs
-            black --check gym-unity
+            git ls-files ml-agents      | xargs pre-commit run --show-diff-on-failure --files
```

Comment:
I don't love this because it makes it harder to run the same checks on CI and locally. After mypy checks are running (in another PR) I'll revisit this.

Reply:
Figured out how to get mypy working with a single invocation, so can do this in one call now.